scheduler-server/scheduling-dsl-compiler/package.json

```diff
@@ -1,13 +1,12 @@
 {
   "name": "scheduling-dsl-compiler",
-  "version": "1.0.0",
```

Comment:
Why is `version` being removed?

Reply:
We don't want version information held in the package files - they are misleading at best.